src/sentry/features/__init__.py

```diff
@@ -4,35 +4,80 @@
 from .handler import *  # NOQA
 from .manager import *  # NOQA
 
+# The feature flag system provides a way to turn on or off features of Sentry.
+#
+# Registering a new feature:
+#
+# - Determine what scope your feature falls under. By convention we have a
+#   organizations and project scope which map to the OrganizationFeature and
+#   ProjectFeature feature objects. Scoping will provide the feature with
+#   context.
+#
+#   Organization and Project scoped features will automatically be added into
+#   the Organization and Project serialized representations.
+#
+#   Additional feature contexts can be found under the features.base module,
+#   but you will typically deal with the organization or project.
+#
+#   NOTE: There is no currently established convention for features which do not
+#         fall under these scopes. Use your best judgment for these.
```

Comment:
Do we need more types than global, organization and project? Perhaps we should recommend folks not make more kinds.